grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -320,14 +320,17 @@ class InventoryItemController {
         requisitionItems = requisitionService.getPendingRequisitionItems(location, product)
         requisitionItems.groupBy { it.requisition }.collect { k, v ->
             itemsMap.put(k, [
+                    picklistItemsByLot: v.get(0).requisition.picklist ?
```

Comment:
I think you can use k instead of v.get(0).requisition,
also you don't need a null check you can use k?.picklist.? (it's null safe)